var/spack/repos/builtin/packages/py-beautifulsoup4/package.py

```diff
@@ -33,6 +33,9 @@ class PyBeautifulsoup4(PythonPackage):
     homepage = "https://www.crummy.com/software/BeautifulSoup"
     url = "https://pypi.python.org/packages/source/b/beautifulsoup4/beautifulsoup4-4.4.1.tar.gz"
```

Comment:
Can you actually convert this to `pypy.io` instead and see if you can remove the version-specific URLs below?